monai/networks/layers/factories.py

```diff
@@ -103,17 +95,6 @@ def add_factory_callable(self, name: str, func: Callable) -> None:
             + ".\nPlease see :py:class:`monai.networks.layers.split_args` for additional args parsing."
         )
 
-    def factory_function(self, name: str) -> Callable:
```

Comment:
I would also leave this method here if the class inherits from `ComponentStore` and leave it's name so that it's clearer what it is decorating.